docs/benchmarks/benchmark.sh

```
@@ -0,0 +1,200 @@
+  done
+
+  # Sort times and get min/median/max
+  IFS=$'\n' sorted=($(sort -n <<<"${times[*]}"))
+  min="${sorted[0]}"
+  median="${sorted[1]}"
+  max="${sorted[2]}"
+
+  echo "  Times: min=${min}s, median=${median}s, max=${max}s" | tee -a "$RESULTS"
+
+  # Count panic points (for verification)
+  COUNT=$((cd jonesy && ../target/release/jonesy --bin "$abs_binary" 2>/dev/null) | \
```

Shellcheck (0.11.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)